adapters/kagome/src/host_api/helpers.cpp

```diff
@@ -35,8 +35,6 @@
 
 #include <kagome/host_api/impl/host_api_factory_impl.hpp>
 
-#include <kagome/offchain/impl/offchain_persistent_storage.hpp>
-
```

Comment:
Please leave this header included. We do need this header. I was just asking you to remove the redundant code you added to initialize an additional persistent offchain storage that is not used. We do already create one [here](https://github.com/w3f/polkadot-tests/blob/main/adapters/kagome/src/host_api/helpers.cpp#L190-L192)